packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/types/ImageHtmlOptions.ts

```diff
@@ -19,4 +21,22 @@ export default interface ImageHtmlOptions {
      * Background color of the rotate handle
      */
     rotateHandleBackColor: string;
+
+    /**
+     * Image editing data
+     */
+    editInfo: ImageEditInfo;
+
+    /**
+     * The number of handlers should be adapted to the image size
```

Comment:
The comment says it is a number, but the actual type is a boolean. Which is right?